streams/src/main/java/org/apache/kafka/streams/processor/StateStore.java

```diff
@@ -58,4 +58,10 @@
      * @return  {@code true} if the storage is persistent&mdash;{@code false} otherwise
      */
     boolean persistent();
```

Comment:
I should probably rename `isOpen` to `open`. The style seems to be to drop the `get` and `is` prefixes